packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxDotAddresses/selectors.ts

```diff
@@ -0,0 +1,67 @@
+// @ts-ignore
+import { concat, curry, reduce, sequence } from 'ramda'
+
+import { Exchange, Remote } from 'blockchain-wallet-v4/src'
+import { ADDRESS_TYPES } from 'blockchain-wallet-v4/src/redux/payment/btc/utils'
+import { selectors } from 'data'
+
+export const getData = (
+  state,
+  ownProps: {
+    exclude?: Array<string>
+    includeCustodial?: boolean
+    includeExchangeAddress?: boolean
+  }
+) => {
+  const {
+    /* exclude = [], */ includeCustodial,
```

Comment:
need to keep the `exclude` or can it be deleted?

Reply:
done